Cargo.toml

```diff
@@ -72,7 +74,7 @@ default = ["full","packagesigning", "sqlite-bundled"]
 sqlite-bundled = ["rusqlite/bundled"]
 telemetry = ["sentry"]
 update-notifications= ["billboard", "colored"]
-prehash-module = ["hex", "blake3"]
+prehash-module = ["blake3"]
```

Comment:
Are we making hex mandatory because we want to start removing the extra feature flags from last week other discussion?

Reply:
No, this was an accident when I tried debugging the SignedUrl I needed hex, I just forgot to undo this when I resolved the issue. Thanks for noticing.